engine/src/test/java/io/camunda/zeebe/engine/state/migration/to_1_1/LegacyDbMessageSubscriptionState.java

```diff
@@ -94,10 +94,10 @@ public void put(final long key, final MessageSubscriptionRecord record) {
 
     messageSubscription.setKey(key).setRecord(record).setCommandSentTime(0);
 
-    subscriptionColumnFamily.put(elementKeyAndMessageName, messageSubscription);
+    subscriptionColumnFamily.upsert(elementKeyAndMessageName, messageSubscription);
```

Comment:
:thinking: Hmm, now we get into philosophical territory. This class is a verbatim copy of the old `DbMessageSubscriptionState` anno 1.0.0. It was used to restore an old state with the old implementation, then run and test the migration.

Now the old code is changed to fit to the new API. Which is necessary and pragmatic. You also modified it in a way that is semantically the same. Philipp warned me that this day will come.